### PR TITLE
Add main repository

### PR DESCRIPTION
**Pull request by marcmo**

### CODE REVIEW

File: otterdog/eclipse-openbsw.jsonnet [hunk 1]
```diff
@@ -9,4 +9,15 @@ orgs.newOrg('eclipse-openbsw') {
       actions_can_approve_pull_request_reviews: false,
     },
   },
+  _repositories+:: [
+    orgs.newRepo('openbsw') {
+      private: true,
```
Comment (netomi) [inline otterdog/eclipse-openbsw.jsonnet:14]: is this intentional? this would create a private repo.

Comment (marcmo) [reply otterdog/eclipse-openbsw.jsonnet:14]: yes...intentional until we have uploaded all code and have a valid gh-pages documentation page in place

Comment (netomi) [reply otterdog/eclipse-openbsw.jsonnet:14]: then add `allow_forking: false` as property or `members_can_fork_private_repositories: true` to the org settings if this repo should be forkable.

Comment (netomi) [reply otterdog/eclipse-openbsw.jsonnet:14]: tbh, its the first time I see that people are willing to create a private repo. Its certainly not needed imho.

**Comment by mueller-o** [comment]
commit done with wrong github user, will be dropped

**Pull request closed without merging**